lib/oauthdb/utils.js

```diff
@@ -23,6 +24,10 @@ module.exports = {
     switch (err.errno) {
     case 101:
       return error.unknownClientId(err.clientId)
+    case 108:
+      return oauthErrors.invalidToken()
```

Comment:
Are you just propagating the same errno? [108 is a missing parameter error on the auth-server](https://github.com/mozilla/fxa-auth-server/blob/0b02308873aaa114f7db2a1b0eacc20d3a6d9426/lib/error.js#L19) and [116 is an endpoint not supported error on the auth server](https://github.com/mozilla/fxa-auth-server/blob/0b02308873aaa114f7db2a1b0eacc20d3a6d9426/lib/error.js#L27), the content server will see those errnos and show the user an incorrect error message. If the numbers are reused, can you use new error numbers here?

Reply:
Good catch, gonna clean it up so it's not duplicating errnos!